htdocs/core/class/html.form.class.php

```diff
@@ -1435,8 +1435,8 @@ function select_dolusers($selected='', $htmlname='userid', $show_empty=0, $exclu
        {
         	if (! empty($conf->multicompany->transverse_mode))
         	{
-        		$sql.= ", ".MAIN_DB_PREFIX."usergroup_user as ug";
-        		$sql.= " WHERE ug.fk_user = u.rowid";
+        		$sql.= " LEFT JOIN ".MAIN_DB_PREFIX."usergroup_user as ug";
+        		$sql.= " ON ug.fk_user = u.rowid";
```

Comment:
@eldy Exact. This is the point if a user is not in a group it not selectable in select in transverse mode. Just update the PR